muninforAndroid/src/main/res/values/strings.xml

```diff
@@ -269,15 +271,14 @@
 	<string name="disableHDGraphs">Disable HD Graphs</string>
 	<string name="downloadError">Download error</string>
 	<string name="certificate_error">Your connection is not private</string>
-	<string name="alert_opensource">Did you know? Munin for Android is now open-source!</string>
+	<string name="alert_opensource">Munin for Android is now GPLv2 copyleft libre software.</string>
```

Comment:
The previous sentence sounds better, don't you think?

Reply:
It seems like overacting to me.

Reply:
+1.
We may remove alert soon since it's been in place for a bunch of time